cmd/deployment_deleter.go

```diff
@@ -100,6 +100,14 @@ func (c *deploymentDeleter) DeleteDeployment(stage biui.Stage) (err error) {
 		return bosherr.WrapError(err, "Loading deployment state")
 	}
 
+	if deploymentState.CurrentIP != "" {
```

Comment:
i dont think we need to clear out current ip during deletion since we are deleting everything.